core/electron/src/backend/ElectronHost.ts

```diff
@@ -57,11 +56,6 @@ export interface ElectronHostOptions {
   rpcInterfaces?: RpcInterfaceDefinition[];
   /** list of [IpcHandler]($common) classes to register */
   ipcHandlers?: (typeof IpcHandler)[];
-  /** if present, [[NativeHost.authorizationClient]] will be set to an instance of NativeAppAuthorizationBackend and will be initialized. */
-  authConfig?: NativeAppAuthorizationConfiguration;
```

Comment:
I see this is removed here but a handful of the test-apps are still using it. Can you check that out?

Reply:
Just removed them - they were left over and not being used since auth config variables are now directly used when creating the `ElectronAuthorizationBackend`. The only references of 'authConfig' left are the ones being used by `MobileHostOpts` which do seem to still be in use.

Reply:
okay, cool sounds good.